doc/changelog.rst

```diff
@@ -100,6 +100,9 @@ Breaking Changes in 4.0
 - Removed :class:`bson.binary.UUIDLegacy`.
 - The "tls" install extra is no longer necessary or supported and will be
   ignored by pip.
+- ``tz_aware``, a keyword argument for :class:`~pymongo.mongo_client
```

Comment:
This line is incorrect. The tz_aware for MongoClient always defaulted to false. Instead you should say that:
The tz_aware argument for JSONOptions defaults to False instead of True. `json_util.loads` now decodes datetime as naive by default.

Reply:
Done.